pyhafas/profile/db/__init__.py

```diff
@@ -1,58 +1,85 @@
 import pytz
 
-from pyhafas.profile.base import BaseProfile
+from pyhafas.profile.base.helper.date_time import BaseDateTimeHelper
+from pyhafas.profile.base.helper.format_products_filter import (
+    BaseFormatProductsFilterHelper,
+)
+from pyhafas.profile.base.helper.parse_leg import BaseParseLegHelper
+from pyhafas.profile.base.helper.parse_lid import BaseParseLidHelper
+from pyhafas.profile.base.helper.parse_remark import BaseParseRemarkHelper
+from pyhafas.profile.base.requests.journey import BaseJourneyRequest
+from pyhafas.profile.base.requests.journeys import BaseJourneysRequest
+from pyhafas.profile.base.requests.location import BaseLocationRequest
+from pyhafas.profile.base.requests.station_board import BaseStationBoardRequest
+from pyhafas.profile.base.requests.trip import BaseTripRequest
+from pyhafas.profile.db.helper.request import DBRequestHelper
+from pyhafas.profile.interfaces import ProfileInterface
 
 
-class DBProfile(BaseProfile):
+class DBProfile(
+    DBRequestHelper,
+    BaseFormatProductsFilterHelper,
+    BaseParseLidHelper,
+    BaseDateTimeHelper,
+    BaseParseLegHelper,
+    BaseParseRemarkHelper,
+    BaseLocationRequest,
+    BaseJourneyRequest,
+    BaseJourneysRequest,
+    BaseStationBoardRequest,
+    BaseTripRequest,
+    ProfileInterface,
+):
     """
     Profile of the HaFAS of Deutsche Bahn (DB) - German Railway - Regional and long-distance trains throughout Germany
     """
+
+    addMicMac: bool = False
```

Comment:
this is already False in the BaseProfile

Reply:
Yes, but if I remove this, I get the following error. 

```
AttributeError: 'DBProfile' object has no attribute 'addMicMac'
```

Reply:
You should be able to just write
```diff
-class DBProfile(BaseProfile):
+class DBProfile(
+    DBRequestHelper,
+    BaseProfile
+):
```
but this is also not necessary anymore when we include the retry feature in all profiles.